src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java

```diff
@@ -79,22 +78,34 @@ private void createPanel() {
     layout = new CardLayout();
     panel = new JPanel(layout);
 
+    var notConnectedCard = new JPanel(new GridBagLayout());
+    var connectedCardWrapper = new JPanel(new GridBagLayout());
+
+    var helpLabelConnected = HelpLabelUtils.currentFileHelp();
+    var helpLabelNotConnected = HelpLabelUtils.currentFileHelpNotConnected();
+
+    var gce = new GridBagConstraints(GridBagConstraints.EAST, 0, 1, 1, 0, 0, GridBagConstraints.EAST, GridBagConstraints.NONE,
+      JBUI.insets(2, 20, 2, 0), 0, 0);
+
+    var helpLabelTextConnected = new JLabel("What's in this view");
+    var helpLabelTextNotConnected = new JLabel("What's in this view");
+
+    notConnectedCard.add(helpLabelTextNotConnected);
+    notConnectedCard.add(helpLabelNotConnected);
+    notConnectedCard.add(new NotConnectedCard(), gce);
+
     connectedCard = new ConnectedCard();
+    connectedCardWrapper.add(helpLabelTextConnected);
+    connectedCardWrapper.add(helpLabelConnected);
+    connectedCardWrapper.add(connectedCard, gce);
 
     panel.add(new EmptyCard(), EMPTY);
     panel.add(new NotConnectedCard(), NOT_CONNECTED);
-    panel.add(connectedCard, CONNECTED);
+    panel.add(notConnectedCard, NOT_CONNECTED);
```

Comment:
Those 2 lines seem a bit strange, it gives the impression that we are adding 2 times the same thing